pkg/api/daemon.go

```diff
@@ -53,10 +56,31 @@ func Run(context *clusterd.Context, c *Config) error {
 		return fmt.Errorf("failed to write connection config. %+v", err)
 	}
 
+	CephMonConfWatcher(context, c)
 	ServeRoutes(context, c)
 	return nil
 }
 
+func CephMonConfWatcher(context *clusterd.Context, c *Config) {
+	go func() {
+		w, err := context.Clientset.CoreV1().ConfigMaps(c.namespace).Watch(metav1.ListOptions{})
```

Comment:
could you add a comment about why we need to watch it?

Reply:
This will watch all configmaps on that namespace. Since we only care about about the mon configmap, I would prefer to watch that instead.